containerd.ps1

```diff
@@ -0,0 +1,29 @@
+# Source: https://github.com/containerd/containerd/blob/main/docs/getting-started.md#installing-containerd-on-windows
+# If containerd previously installed run:
+Stop-Service containerd
```

Comment:
Do the runners run jobs in parallel? What happens if these installation scripts are run simultaneously on the same runner?